test/dynamo/test_autograd_function.py

```diff
@@ -271,7 +271,9 @@ def test_print_in_bwd(self):
         model = CustomFuncBwdPrintModule()
         opt_model = torch.compile(model, backend="eager", fullgraph=True)
         x = torch.randn(2, 2, dtype=torch.double, requires_grad=True)
-        with self.assertRaisesRegex(torch._dynamo.exc.Unsupported, "builtin: print"):
+        with self.assertRaisesRegex(
+            torch._dynamo.exc.Unsupported, "Failed to trace builtin operator"
```

Comment:
Jansel says: Make sure this has the operator name in it (though it's possible that this is just part of it)